test/src/main/java/org/apache/rocketmq/test/util/RandomUtil.java

```diff
@@ -116,17 +116,17 @@ private static String getString(int n, int arg[]) {
     }
 
     public static String getStringWithCharacter(int n) {
-        int arg[] = new int[] {'a', 'z' + 1, 'A', 'Z' + 1};
+        int[] arg = new int[]{'a', 'z' + 1, 'A', 'Z' + 1};
```

Comment:
Thanks for the optimization. I think space is need before curly braces “{”.

Reply:
yes, it's better.